### PR TITLE
Test: Commit URL context

### PR DESCRIPTION
**Pull request by peterjuma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,116 +13,3 @@ while (i < 10) {
...
-
-
-## Not the God question
-
-
-### Editing file on 2022-01-04_11-20-30
-
-> This is a blockquote.
->
-> This is the second paragraph in the blockquote.
```
Comment (peterjuma) [inline README.md:25]: Start here

**Review by jumapeter — APPROVED** [review]

**Comment by peterjuma** [comment]
cf1782aca3e1310b7f46f582f7b3a67770d6f32d

**Comment by peterjuma** [comment]
https://github.com/peterjuma/demo/projects/2

**Comment by peterjuma** [comment]
Fixes #51
Fixes #52
Fixes #53
Fixes #54
Fixes #55
Fixes #56
Fixes #57
Fixes #58
Fixes #59
Fixes #60
Fixes #61
Fixes #62
Fixes #63
Fixes #64
Fixes #65
Fixes #66
Fixes #67
Fixes #68
Fixes #69
Fixes #70

**Pull request merged**